### PR TITLE
MVP: Outbound link tracking

### PR DESCRIPTION
**Pull request by schwers**

### CODE REVIEW

File: src/app/components/OutboundLink/index.jsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+  // expected props from connect
+  userId: T.string,
+};
+
+const selector = createSelector(
+  state => {
+    const { user, accounts } = state;
+    const userAccount = user.loggedOut ? null : accounts[user.name];
+    return userAccount ? convertId(userAccount.id) : null;
```
Comment (phil303) [inline src/app/components/OutboundLink/index.jsx]: love this specific selector

File: src/app/components/OutboundLink/index.jsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+      onMouseDown={ e => {
+        // don't show the redirect url for right-click context menus
+        if (e.which === 3) {
+          return true;
+        }
+
+        setOutboundURL(e.target, outboundLink, userId);
+      } }
+      onMouseLeave={ e => resetOriginalURL(e.target, href) }
+      onTouchStart={ e => setOutboundURL(e.target, href) }
```
Comment (phil303) [inline src/app/components/OutboundLink/index.jsx]: :no_bell: i'm curious, while dev'ing this, did you have any problems with both the mouse down and the touch start event firing?

**Comment by phil303** [comment]
seems pretty solid to me

**Comment by nramadas** [comment]
:fish:

**Pull request merged**